src/io/obf.rs

```diff
@@ -17,7 +21,7 @@ where
     M: PolyMatrix,
     SU: PolyUniformSampler<M = M>,
     SH: PolyHashSampler<[u8; 32], M = M>,
-    ST: PolyTrapdoorSampler<M = M>,
+    ST: PolyTrapdoorSampler<M = M> + Send + Sync,
```

Comment:
Can you add `Send + Sync` to the `PolyTrapdoorSampler` trait definition instead of requiring them in this function?